lib/evm_database.rb

```diff
@@ -16,6 +16,12 @@ class EvmDatabase
     MiqReport
     RssFeed
     MiqWidget
+    MiqAction
+    MiqEventDefinition
+    Condition
+    MiqPolicy
+    MiqPolicyContent
+    MiqPolicySet
```

Comment:
Why are these primordial?  primordial are only for things that the system can't even boot without (and yes, by that sentiment I agree that RssFeed and VmdbDatabase should not be in there :smile:)

FYI, Non-primoridal will still get seeded, even if not in this list